modules/print-styles.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
+
+var allPrintStyles = {};
+var listeners = [];
+
+var addListener = function (listener: Function): void {
```

Comment:
Fair enough. Though I still think the subscribe method returning an object with remove is a better API than addListener/removeListener